snakemake/Snakefile

```diff
@@ -59,8 +59,8 @@ rule init_db:
     conda: "env.yaml"
     output: f"{OUTPUT_DIR}/job_status/init_db.txt"
     shell: """
-    touch {output}
     guidescanpy init-db
+    touch {output}
```

Comment:
"touching" files that are meant only to keep track of snakemake rules should happen as the last step, not the first, since otherwise we will not be able to keep track of steps that fail (The `touch` command should only be reached if all else worked fine).

Reply:
This makes sense. Thank you!